server/routes/activities/unlock-list/handlers/plannedEvents.ts

```diff
@@ -15,16 +15,16 @@ export default class PlannedEventsRoutes {
   GET = async (req: Request, res: Response): Promise<void> => {
     const { user } = res.locals
     const { date } = req.query
-    const { location, timeSlot } = req.session.unlockListJourney
+    const { locationKey, timeSlot } = req.session.unlockListJourney
 
-    const subLocations = await this.activitiesService
+    const location = await this.activitiesService
       .getLocationGroups(user)
-      .then(locations => locations.find(loc => loc.key === location).children.map(loc => loc.key))
+      .then(locations => locations.find(loc => loc.key === locationKey))
 
```

Comment:
Does this change the logic slightly? It was pulling all sublocations of the main location, and now pulls the location?